### ISSUE TITLE
Compiling packages with Fortran

### ISSUE THREAD

**Issue by WastlM** [jeroen/solarisvm#2]
Installing packages requiring Fortran compilation (package "deldir" in my case) did not work out of the box using the currently provided VMware image. It would be helpful to include instructions on how to setup a Fortran compiler in the readme.md. This is what I did:

```
## install a Fortran compiler
/opt/csw/bin/pkgutil -y -i gcc4gfortran 

## Include CSW in the PATH (for R)
export PATH=/opt/csw/bin:/usr/ccs/bin:$PATH

## inspect current R configuration (we need an additional PATH for make)
export PATH=/usr/ccs/bin:$PATH
R CMD config FC
## reports /opt/csw/bin/gfortran-5.2, which does not exist

## set newly installed Fortran compiler in Makevars.site
echo FC=gfortran > /opt/csw/lib/R/etc/Makevars.site
```

**Comment by jeroen** [comment]
Is `/opt/csw/bin/gfortran-5.2` included with `gcc5gfortran`?

**Comment by WastlM** [comment]
Yes, thanks! The `gcc5gfortran` catalog is described at https://www.opencsw.org/packages/CSWgcc5gfortran/, which states

> Version: 5.2.0,REV=2015.07.31

Installing that version adds /opt/csw/bin/gfortran-5.2 (as well as a `gfortran` symbolic link to that file), so changing the FC environment variable via a Makevars file is not necessary.

**Comment by jeroen** [comment]
OK so I will preinstall `gcc5gfortran` on the next version of the vm (if ever).
